lametro/events.py

```diff
@@ -105,5 +192,33 @@ def scrape(self, window=None) :
                     e.add_source(web_event['Meeting Details']['url'], note='web')
                 else:
                     e.add_source('https://metro.legistar.com/Calendar.aspx', note='web')
-
             yield e
+            
+
+class APIEvent(dict):
```

Comment:
I might also call this `LAMetroAPIEvent`. Can you say more about why this inherits from `dict`, perhaps in a docstring?

Reply:
I'm basically just adding methods to existing dict.

Reply:
This class is sort of removed from the scraper logic that yields API events. It would be nice to add a docstring saying we use this class to wrap API event dicts yielded from the scraper, to add some convenience methods & properties.

Reply:
added a docstring